### ISSUE TITLE
sequenceDiagram autonumbers don't align when labels contain <br>

### ISSUE THREAD

**Issue by deftdawg** [mermaid-js/mermaid#1405]
If a line break `<br>` is present in the label of an element with `autonumber` enabled the number value is rendered separately from the circle that would typically contain it:

```mermaid
sequenceDiagram
    autonumber
    Alice->>+John: Hello John,<br> how are you?
    Alice->>+John: John, can you hear me?
```

See **1** Below:

![image](https://user-images.githubusercontent.com/4991612/81634294-ab0c0000-93dc-11ea-9e47-35489bef65e7.png)

**Comment by GDFaber** [comment]
Hi @deftdawg, this seems to be fixed in release 8.5.1 which just came out. I tried the example and it worked properly. Can you confirm that?

**Comment by deftdawg** [comment]
@GDFaber Yes thanks!  It's fixed, just tried it with mermaid 8.5.1 in the live editor.  Looks good!

![image](https://user-images.githubusercontent.com/4991612/82155642-60f3a600-9844-11ea-8346-298fe1b716cb.png)
